pkg/cvo/cvo_scenarios_test.go

```diff
@@ -1471,59 +1633,47 @@ func TestCVO_UpgradeUnverifiedPayloadRetrieveOnce(t *testing.T) {
 			t.Fatalf("saw too many sync events of the wrong form")
 		}
 	}
+	// wait until the new payload is applied
+	count = 0
+	for {
+		var status SyncWorkerStatus
+		select {
+		case status = <-worker.StatusCh():
+		case <-time.After(3 * time.Second):
+			t.Fatalf("never saw expected apply event")
+		}
+		if status.loadPayloadStatus.Step == "PayloadLoaded" {
+			break
+		}
+		t.Log("Waiting to see step PayloadLoaded")
+		count++
+		if count > 8 {
+			t.Fatalf("saw too many sync events of the wrong form")
+		}
+	}
 	verifyAllStatus(t, worker.StatusCh(),
 		SyncWorkerStatus{
 			Total:       3,
-			Step:        "ApplyResources",
 			VersionHash: "DL-FFQ2Uem8=",
 			Actual: configv1.Release{
 				Version: "1.0.1-abc",
 				Image:   "image/image:1",
 				URL:     configv1.URL("https://example.com/v1.0.1-abc"),
 			},
+			//LastProgress: time.Unix(1, 0),
```

Comment:
nit: another commented-out line to drop?

Reply:
https://github.com/openshift/cluster-version-operator/pull/745